src/components/radios/template.njk

```diff
@@ -2,44 +2,46 @@
 {% from "label/macro.njk" import govukLabel %}
 
 {#- innerHtml -> captures the HTML so we can optionally nest it in a fieldset -#}
-<div class="govuk-c-radios
-{%- if params.classes %} {{ params.classes }}{% endif %}"{% for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
-  {%- set innerHtml %}
-  {% for item in params.items %}
-  {% set idPrefix = params.idPrefix if params.idPrefix else params.name %}
-  {% set id = item.id if item.id else idPrefix + "-" + loop.index %}
-  <div class="govuk-c-radios__item">
-    <input class="govuk-c-radios__input" id="{{ id }}" name="{{ params.name }}" type="radio" value="{{ item.value }}"
-    {{-" checked" if item.checked }}
-    {{-" disabled" if item.disabled }}>
-    {{ govukLabel({
-      html: item.html,
-      text: item.text,
-      classes: 'govuk-c-radios__label',
-      attributes: item.label.attributes,
-      for: id
-    }) | indent(4) | trim  }}
-  </div>
-  {% endfor %}
-  {% endset %}
+<div class="govuk-o-form-group {%- if params.errorMessage %} govuk-o-form-group--error{% endif %}">
```

Comment:
I don't think we need this wrapper anymore since this can be applied with the fieldset.

This applies to any components that use a wrapping fieldset.

<img width="501" alt="screen shot 2018-02-16 at 12 55 47" src="https://user-images.githubusercontent.com/2445413/36308464-c6e7db88-1318-11e8-85ce-4e89bb6bee2a.png">

Reply:
Thanks @nickcolley, that's now fixed.